internal/healthchecks/healthchecks_test.go

```diff
@@ -148,15 +149,16 @@ func (c MultipleSuccessResultCheck) Name() string {
 	return "MultipleResult Check"
 }
 
-func (c MultipleSuccessResultCheck) RunCheck(logger *log.Logger) error {
+func (c MultipleSuccessResultCheck) RunCheck(logger *logs.FileLogger) error {
 	return errors.Join(nil, nil, nil)
 }
 
 func TestMultipleSuccessResultCheck(t *testing.T) {
 	sCheck := MultipleSuccessResultCheck{}
 	expectedSuccess := generateExpectedResultMessage(sCheck.Name(), "PASS")
 	fmt.Println(expectedSuccess)
```

Comment:
Could you please remove this line ? Left it there by mistake in the PR where I add this unit tests.

Reply:
Done!